pkg/kv/kvserver/loqrecovery/loqrecoverypb/recovery.proto

```diff
@@ -167,3 +167,31 @@ message NodeRecoveryStatus {
   // aggregated error messages containing all encountered errors.
   string error = 5;
 }
+
+// PlanApplicationResult is a value stored inside node local storage whenever
+// loss of quorum recovery plan is applied for the purpose of tracking recovery
+// progress.
+message PlanApplicationResult {
+  bytes applied_plan_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customname) = "AppliedPlanID",
+    (gogoproto.customtype) = "github.com/cockroachdb/cockroach/pkg/util/uuid.UUID"];
+  // LastProcessingTime is a node wall clock time when last recovery plan was applied.
+  google.protobuf.Timestamp apply_timestamp = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true];
+  // If most recent recovery plan application failed, Error will contain
+  // aggregated error messages containing all encountered errors.
```

Comment:
It is possible to have more than one commit error in case we have multiple stores and batch commit fails. At this point we collect all commit errors since we can't rollback anymore.